modelserver/model_worker.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from concurrent.futures import ProcessPoolExecutor
+import logging
+import os
+import queue
+import multiprocessing as M
+from llama_cpp import Llama
+from typing import AsyncGenerator
+
+from modelserver.types.api import CompletionInferenceRequest
+
+"""
```

Comment:
documenting b/c while this code looks simple, it took many hours today to figure out lol